app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  def update_post_counter
+    users.posts_counter += 1
+  end
```

Comment:
## Optional
- [ ] I highly suggest that you use [Rails active callback](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the #update_post_counter method every time a new post has been saved to the database, then make the `update_post_counter` method private